src/app/portlets/dot-edit-page/shared/models/layout-body.model.ts

```diff
@@ -0,0 +1,5 @@
+import { LayoutRow } from './layout-row.model';
+
+export interface LayoutBody {
```

Comment:
Let's add ```Dot``` prefix to all our interfaces as a convention, even when we are matching it with the ones in the backend.

Thoughts? @luisrojascr @freddyucv

Reply:
really a don't like prefix in interfaces name, but is not a big deal really

Reply:
I think we need doc in this interface: 

"
  Represents the body of a Layout Object
 @see Layout
"

I don't really know if we have something like @see in tsdoc but...

Reply:
My concern is because sometimes we need to create a interface with the name of a interface already exist like: "MenuItem" that exist in PrimeNG and I have to create one for DotCMS.